.github/workflows/python-validate.yml

```diff
@@ -90,6 +85,10 @@ jobs:
         if: steps.pathChanges.outputs.sdk == 'true'
         run: |
           cd python/packages/sdk
+          python3 -m venv .venv
```

Comment:
If that's needed let's propose it with different PR

Reply:
Sure, I've created https://github.com/serverless/console/pull/473

I'll clean up this one now.